sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -226,9 +226,15 @@
 
   /**
    * @since 2.14
+   * @deprecated since 7.1, this setting should not be used by plugin
    */
+  @Deprecated
   String CORE_AUTHENTICATOR_REALM = "sonar.security.realm";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Do not forget to remove this deprecated code someday. ([squid:S1133](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjl%2FSONAR-10300%2Fforbid_system_properties_in_settings_ws&issues=AWFNBHY4NbtQu1yJ8xlZ&open=AWFNBHY4NbtQu1yJ8xlZ)

[ISSUE_KEY]: AWFNBHY4NbtQu1yJ8xlZ